.github/workflows/linters.yml

```diff
@@ -3,6 +3,19 @@ on: [workflow_call]
 permissions: {}
 
 jobs:
+  uniffi-bindings:
+      runs-on: ubuntu-22.04
```

Comment:
can we run this on all 3 supported desktop OS types to make sure it stays working everywhere?